service/src/main/java/com/codeforcommunity/requester/Emailer.java

```diff
@@ -23,6 +23,8 @@ public class Emailer {
       PropertiesLoader.loadProperty("email_subject_account_deleted");
   private final String subjectEmailNeighborhoods =
       PropertiesLoader.loadProperty("email_subject_neighborhood_notification");
+  private final String subjectImageRejected =
+      PropertiesLoader.loadProperty("email_subject_image_rejected");
```

Comment:
If we want to load the email subject this way, we'll want to add this to `server.properties.example` - you can simply match what we do for the other email subjects. 

When the backend gets deployed to AWS, it'll copy the property name and value into the `server.properties` and be available here; if you're really interested you can take a look at `deploy.py` for more details

Reply:
Thanks for the info, good catch! Ill add it